go/cmd/vtctldclient/command/tablets.go

```diff
@@ -190,6 +192,15 @@ Note that, in the SleepTablet implementation, the value should be positively-sig
 		Args:                  cobra.ExactArgs(1),
 		RunE:                  commandStopReplication,
 	}
+	// GetPermissions asks the remote tablet for its permissions list.
+	GetPermissions = &cobra.Command{
+		Use:                   "GetPermissions <tablet_alias>",
+		Aliases:               []string{"GetPermissions"},
```

Comment:
This is not necessary, since you already have `GetPermissions` in the Use ([c.f.](https://pkg.go.dev/github.com/spf13/cobra#Command)

Reply:
are you talking about line 195 here ?

Reply:
You don't need Aliases, since your only alias is the Use directly above